ide/git/src/org/netbeans/modules/git/ui/push/PushBranchesStep.java

```diff
@@ -98,6 +98,13 @@ public HelpCtx getHelp() {
         return new HelpCtx(PushBranchesStep.class);
     }
 
+    /**
+     * *
+     * @param cfg Congiruration of the remote repository including urls of
```

Comment:
Please keep it on one line and fix typo: "Configuration of the remote repository including urls of remote"